app/lib/constants/settings/general.rb

```diff
@@ -0,0 +1,146 @@
+module Constants
+  module Settings
+    module General
+      IMAGE_PLACEHOLDER = "https://url/image.png".freeze
+      SVG_PLACEHOLDER = "<svg ...></svg>".freeze
+
+      DETAILS = {
+        credit_prices_in_cents: {
+          small: {
+            description: "Price for small credit purchase (<10 credits).",
```

Comment:
Suggestion: we might consider extracting these strings into an internationalization file in the future. Completely out of scope for this PR, and I think it would need an RFC to really address it.

Reply:
Preaching to the choir here, I've long been advocating for always doing this **from the beginning** in **every** Rails app. Even if you never add another language it makes your tests less brittle because you can now assert on I18n keys instead of snippets of copy.